lightningd/options.c

```diff
@@ -363,7 +359,9 @@ static void config_register_opts(struct lightningd *ld)
 	opt_register_arg("--tor-service-password", opt_add_tor_service_password, NULL,
 			ld,"Set a Tor hidden service password");
 	opt_register_arg("--tor-auto-listen", opt_set_bool_arg, opt_show_bool,
-			&ld->config.tor_enable_auto_hidden_service ,"Generate and use a temp auto hidden-service and show the onion address");
+			&ld->config.tor_enable_auto_hidden_service , "Generate and use a temp auto hidden-service and show the onion address");
+	opt_register_arg("--allways-use-tor-proxy", opt_set_bool_arg, opt_show_bool,
+			&ld->use_tor_proxy_allways , "Use the Tor propxy allways");
```

Comment:
`--allways-use-tor-proxy` should probably be `--always-use-tor-proxy`

Reply:
:+1:  Thanks! changed it, as i indicated to @ZmnSCPxj  before IQ's like hell but can not read and write ;-)